torch/ao/sparsity/_mappings.py

```diff
@@ -1,4 +1,5 @@
 import torch
+import torch.ao.nn
```

Comment:
To avoid the circular dependency, we removed the __init__.py, meaning python submodule bindings are gone now. That means that before we can use the module at runtime, we need to have the submodule import to recreate those bindings.